minecraft_servers/cragenetwork/manifest.json

```diff
@@ -0,0 +1,16 @@
+{
+  "server_name": "cragenetwork",
+  "nice_name": "Crage Network",
+  "direct_ip": "laby.cragenetwork.com",

```

Comment:
Server not reachable, either fix so that server is joinable with that ip, too or use the normal play.cragenetwork.com IP.

Reply:
Updated